python/python/stacrs/__init__.py

```diff
@@ -1,16 +1,19 @@
-from . import stacrs
-from .stacrs import (
+import pystac
```

Comment:
It was an intentional decision to _not_ import **pystac** at the top level — I'd like this library to be zero-dependency by default.

Reply:
Oh, sorry, that was meant to be the internal `.pystac`.